loadtest.py

```diff
@@ -75,52 +75,58 @@ def get_payload_and_headers(size, dump_names=None, compressed=False):
     return payload, headers
 
 
-def run_test(name, size, dump_names=None, compressed=False):
-    payload, headers = get_payload_and_headers(size, dump_names=dump_names,
-                                               compressed=compressed)
-    resp = utils.post_crash(URL_SERVER, payload, headers)
-
-    print('%s: HTTP %s' % (name, resp.status_code))
-    # Verify HTTP 200
-    resp.raise_for_status()
-    # Verify the response text contains a CrashID
-    utils.verify_crashid(resp.text)
+async def run_test(name, session, size, dump_names=None, compressed=False):
+    payload, headers = get_payload_and_headers(size, dump_names=dump_names, compressed=compressed)
+    async with session.post(URL_SERVER, headers=headers, data=payload,
+                            compress=False, allow_redirects=True) as resp:
```

Comment:
Need to specify `compress=False` here so that aiohttp doesn't additionally compress the payload and chunk it.